luigi/scheduler.py

```diff
@@ -1328,6 +1328,7 @@ def worker_list(self, include_running=True, **kwargs):
                 started=worker.started,
                 state=worker.state,
                 first_task_display_name=self._first_task_display_name(worker),
+                unread_messages=len(worker.rpc_messages),
```

Comment:
Call this `num_unread_messages`? Or even better, `num_unread_rpc_messages`. Would be nice if a grep-search for `rpc_messages` turned out all relevant bits of code for the RPC messages.